### PR TITLE
20200707 1548 - String Changes for iOS Version 28

### PR DESCRIPTION
**Pull request by nbhasin2**
String Changes for iOS Version 28

### CODE REVIEW

**Comment by nbhasin2** [comment]
@Delphine Made another PR to address issue in the last one.

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -3062,6 +3056,29 @@
...
+        <source>Copied Link from clipboard</source>
+        <note>Copied Link from clipboard displayed</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.GoToCopiedLinkLabelV1">
+        <source>Go to copied link</source>
+        <note>Go to link pasted on the clipboard</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.NewTabButtonLabelV1">
+        <source>New Search</source>
+        <note>Open New Tab button label</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3074]: Should the localization note here be updated to "Open New Search button label"?

Comment (nbhasin2) [reply templates/firefox-ios.xliff:3074]: Ah, basically it is opening new tab but we present it as new search as the address bar is where user searches. 

Would you suggest updating? 🤔

Comment (Delphine) [reply templates/firefox-ios.xliff:3074]: Ah gotcha! No, I think it should be ok as is then. Thank you!

**Pull request merged**